styles.css

```diff
@@ -0,0 +1,294 @@
+* {
```

Comment:
You should not write background-color in css reset because the purpose of a CSS reset is to remove default browser styles so that you can start with a clean slate and apply your own styles as needed.